action.yml

```
@@ -64,3 +72,5 @@ runs:
           ${{ inputs.image }}:${{ inputs.tag }}
           ${{ inputs.extra_tags }}
         labels: ${{ steps.meta.outputs.labels }}
+        cache-from: type=s3,region=eu-central-1,bucket=gutss3dockercache,name=${{ github.event.repository.name }}
```

```suggestion
        cache-from: type=s3,region=eu-central-1,bucket=gutss3dockercache,name=${{ inputs.image }}:${{ inputs.tag }}
```
`inputs.tag` already contains the branch name